src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -187,6 +193,19 @@ const useStyles = makeStyles({
     clipPath: 'inset(50%)',
     whiteSpace: 'nowrap',
   },
+  popoverButtonContainer: {
+    display: 'flex',
+    '& button': {
+      margin: '1rem 1rem 0 0',
+      '&focus': {
+        outline: 'unset',
+      },
+    },
+  },
```

Comment:
We should avoid adding styles to html elements directly. Especially when these are components coming from our Design System components. These styles may conflict with styles already set on the component, and may break in weird ways when the component is updated, f.ex if the Button component was changed to use a `div` instead of `button`, or if it had some `margin` already set. Catching these types of errors are difficult since its not easily testable. (I know this example is pretty far fetched - changing a Button component to be a `div` under the hood is extremely unlikely, but it illustrates the point, and this is a generic problem with overriding styles)

Is the intention to add a gap between the buttons? If thats the case, I would suggest to instead use the `gap` property here on the parent flex container instead (`popoverButtonContainer`).

The margin-top here is probably for spacing between the text and the buttons? That margin should IMO be moved to the `popoverButtonContainer`, and not be on the buttons.

```suggestion
  popoverButtonContainer: {
    display: 'flex',
    gap: '1rem',
    marginTop: '1rem',
  },
```

Also not sure why the focus outline is being unset here, so my suggestion does not cater for that at all.